### PR TITLE
Optimize the main interface layout

### PR DESCRIPTION
**Pull request by floatingrain**

### CODE REVIEW

**Review by rurirei — CHANGES_REQUESTED** [review]
please make sure a clean commits tree

File: AndroidLibV2rayLite/go.mod [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module github.com/2dust/AndroidLibV2rayLite
+
+go 1.14
```
Comment (rurirei) [inline AndroidLibV2rayLite/go.mod:3]: golang 1.15+ required

File: V2rayNG/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,117 +1,94 @@
...
-
-            <android.support.v7.widget.Toolbar
-                android:id="@+id/toolbar"
-                android:layout_width="match_parent"
-                android:layout_height="?attr/actionBarSize"
-                android:background="?attr/colorPrimary"
-                app:popupTheme="@style/AppTheme.PopupOverlay" />
-
-        </android.support.design.widget.AppBarLayout>
+        android:fitsSystemWindows="true">
```
Comment (rurirei) [inline V2rayNG/app/src/main/res/layout/activity_main.xml:8]: do not use this as unnecessary.

**Review by rurirei — CHANGES_REQUESTED** [review]
1. no useless files: `.jar` and more
2. do not provide your library: `.aar`

File: AndroidLibV2rayLite/Process/Escort/escort.go [hunk 3]
```diff
@@ -1,11 +1,9 @@
 package Escort
 
 import (
+	"log"
 	"os"
 	"os/exec"
-	"time"
```
Comment (rurirei) [inline AndroidLibV2rayLite/Process/Escort/escort.go]: did you follow the latest tree as this is outdated?

File: V2rayNG/app/build.gradle [hunk 4]
```diff
@@ -16,8 +16,8 @@ android {
         minSdkVersion 17
         targetSdkVersion Integer.parseInt("$targetSdkVer")
         multiDexEnabled true
-        versionCode 212
-        versionName "1.0.2"
+        versionCode 213
```
Comment (rurirei) [inline V2rayNG/app/build.gradle:19]: do not change build config.

File: V2rayNG/app/release/output-metadata.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (rurirei) [inline V2rayNG/app/release/output-metadata.json:1]: no useless files.

File: V2rayNG/app/src/main/java/com/v2ray/ang/util/OtherTools.java [hunk 6]
```diff
@@ -0,0 +1,154 @@
+package com.v2ray.ang.util;
```
Comment (rurirei) [inline V2rayNG/app/src/main/java/com/v2ray/ang/util/OtherTools.java:1]: do not import other packages unless it is forced be.

File: V2rayNG/app/src/main/kotlin/com/v2ray/ang/ui/MainActivity.kt [hunk 7]
```diff
@@ -1,42 +1,41 @@
 package com.v2ray.ang.ui
```
Comment (rurirei) [inline V2rayNG/app/src/main/kotlin/com/v2ray/ang/ui/MainActivity.kt:2]: do not change UI too much, just optimize its coding style.

File: V2rayNG/app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -4,8 +4,8 @@
         xmlns:tools="http://schemas.android.com/tools"
         android:id="@+id/drawer_layout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:fitsSystemWindows="true">
+        android:fitsSystemWindows="true"
```
Comment (rurirei) [inline V2rayNG/app/src/main/res/layout/activity_main.xml:7]: did you see my reviews? do not use this.

File: V2rayNG/app/src/main/res/menu/menu_main.xml [hunk 9]
```diff
@@ -28,9 +28,8 @@
                 app:showAsAction="never" />
 
             <item
-                android:icon="@drawable/ic_add_white_24dp"
                 android:title="@string/menu_item_import_config_custom"
-                app:showAsAction="always">
+                app:showAsAction="ifRoom">
```
Comment (rurirei) [inline V2rayNG/app/src/main/res/menu/menu_main.xml:32]: `always` recommended for compatibility.

**Review by rurirei — CHANGES_REQUESTED** [review]
thanks for your deep work though, please **accept all reviews here and `force push` to necessary and readable commits tree**, as delete all commits currently here.

you changed much, so make this only one commit cleaned please.

**Comment by floatingrain** [comment]
> 1. no useless files: `.jar` and more
> 2. do not provide your library: `.aar`

为什么不能提交.aar？

File: V2rayNG/app/src/main/kotlin/com/v2ray/ang/ui/MainActivity.kt (hunk 7 above)
Comment (floatingrain) [reply V2rayNG/app/src/main/kotlin/com/v2ray/ang/ui/MainActivity.kt:2]: > do not change UI too much, just optimize its coding style.

仅仅代码风格？主界面的目前布局几乎可以称作一团糟。

**Pull request closed without merging**